src/test/obj_pool_lock/obj_pool_lock.c

```diff
@@ -45,18 +45,16 @@
  * CreateProcess works just fine.
  */
 int
-test_process(const char *path, int sleep)
+test_process(char **path, int sleep)
```

Comment:
This is the third copy of this exact same code. My suggestion would be to move it to unittest.c, and while you are at it, change it so that it automatically retrieves the executable name so that you don't have to do anything else.